sass/lightbox.scss

```diff
@@ -78,7 +79,6 @@ html.lightbox-enabled {
   .lightbox-contents {
     @extend %user-select-none;
 
-    cursor: zoom-out;
```

Comment:
The requirements have changed here. Instead of zooming out when a user clicks on the lightbox image we zoom into the image (displaying it at its natural resolution).